src/Util/OperationTypes.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+/*
+ * This file is part of the API Platform project.
+ *
+ * (c) Kévin Dunglas <dunglas@gmail.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace ApiPlatform\Core\Util;
+
+final class OperationTypes
+{
+    const ITEM = 'item';
```

Comment:
Strings are making making things easier too do:

```
foreach (OperationTypes::TYPES as $operationType) {
+            $attribute = "_api_{$operationType}_operation_name";
```

I think it's fine like this but if we want integers we need to change this now or it might be painful later \o/.